VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -153,7 +154,7 @@ public void Put(Guid manageId, ManagePollUpdateRequest updateRequest)
                         else
                         {
                             oldOptions.Add(option);
-                            removedVotes.AddRange(context.Votes.Where(v => v.OptionId == option.Id).ToList());
+                            removedVotes.AddRange(context.Votes.Where(v => v.Option.Id == option.Id).ToList());
```

Comment:
This might need an .Include(v => v.Option). We've had issues before where not having the Include would pass the tests, but fail on deployment.